explainer.md

```diff
@@ -99,68 +99,89 @@ This aim of this API is to help websites provide an optimal initial experience.
-  type: 'media-source',
-  video: { contentType: "video/webm; codecs=vp09.00.10.08", width: 1280, height: 720,
-           framerate: 24, bitrate: 123456 },
-  audio: { contentType: "audio/webm; codecs=opus" },
-  encryption: { robustness: { audio: "bar", video: "foo" },
-                keySystem: "org.w3.clearkey",
-                initDataType: "keyids",
-                persistentState: "required",
-                sessionTypes: [ "temporarypersistent-usage-record",
-                                "persistent-license" ],
-  },
-}).then(result => {
-  // If the key system isn't supported, the key system doesn't support the
-  // codecs, or there is any other issue, supported will be false.
-  if (!result.supported || !result.smooth)
-    throw Error("Don't play this");
-
-  // This call is only meant to get a MediaSystemAccess object.
-  return navigator.requestMediaKeySystemAccess("org.w3.clearkey", [{
-    audioCapabilities: [ { contentType: "video/webm; codecs=opus",
-                           robustness: "bar" } ],
-    videoCapabilities: [ { contentType: "video/webm; codecs=vp09.00.10.08",
-                           robustness: "foo" } ],
-    initDataTypes: [ "keyids" ],
-    persistentState: "required",
-    sessionTypes: [ "temporarypersistent-usage-record",
-                    "persistent-license" ],
-  }]);
-}).catch(_ => {
-  // Try another format/key system/robustness combination.
-});
+Specifically, `rMKSA` takes a sequence of `MediaKeySystemConfigurations`, ordered by preference. Each entry may contain a sequence of initDataTypes and sequences of audio and video contentTypes with robustness. In the dictionary above, all of these sequences are reduced to single values. 
+
+This is a fundamental difference between the APIs. MediaCapabilities aims to describe the quality (smoothness and power efficiency) of support for a single pair of audio and video streams without making a decision for the caller. Callers should still order media configurations as they do with `rMKSA`, only now they walk the list themselves, calling MediaCapabiliites once for each option. These calls will return immediately with the promises resolving asynchronously.
+
+When a key system configuration is included in the `MediaDecodingConfiguration`, `mediaCapabilities.decodingInfo()` will return a promise containing the usual three booleans (`supported`, `smooth`, and `powerEfficient`) plus a `MediaKeySystemAccess` object whenever `supported = true`. The caller may use the `MediaKeySystemAccess` as they would in traditional EME to request media keys and setup encrypted media playback. This removes the need to call `rMKSA`.
+
+ Here's a sample usage:
+
+```Javascript
+let capabilitiesPromises = []
+
+// Like rMSKA(), orderedOptions is ordered from most -> least wanted.
+for (option in orderedOptions) {
+  capabilitiesPromises.push(navigator.mediaCapabilities.decodingInfo(
+      option.decodingConfig, option.keySystemConfig));
```

Comment:
This doesn't correspond to the API description above.

Reply:
Fixed.